server/modules/arenaImport/service/arenaImport/jobs/recordsImportJob.js

```diff
@@ -70,6 +70,12 @@ export default class RecordsImportJob extends Job {
       }
 
       const recordToStore = this.prepareRecordToStore(record)
+
+      const existingRecord = await RecordManager.fetchRecordByUuid(surveyId, recordUuid, this.tx)
```

Comment:
should we check the updateDate?

Reply:
yes, and it should be checked in the mobile app too, to avoid sending data that won't be imported